mod_test/controllers.py

```diff
@@ -99,6 +99,13 @@ def get_data_for_test(test, title=None) -> Dict[str, Any]:
     """
     if len(test.progress) == 0:
         var_average = 'average_time_' + test.platform.value
+
+        # get average build and prep time.
+        prep_average_key = test.platform.value + '_avg_prep_time'
```

Comment:
The ones above are `name_platform`, now you're switching around... Might be better to stick with one naming scheme :)